Fastpack/FlatPacker.ml

```diff
@@ -854,12 +854,29 @@ let pack ?(cache=Cache.fake ()) (ctx : Context.t) channel =
           total_modules := List.concat [ !total_modules; new_modules; ];
           Lwt.return_unit
   in
+  let mode =
+    Printf.sprintf "var __DEV__ = %s;\n"
+      @@ if ctx.mode = Mode.Development then "true" else "false"
+  in
+  let header = (
```

Comment:
Could be probably in one `match` statement?
```
let header, footer = match ..... -> "", ""
```

Reply:
Cool! Didn't tried this feature yet.